packages/extension-ui/src/Popup/AuthManagement/index.tsx

```diff
@@ -144,6 +144,10 @@ export default styled(AuthManagement)`
   display: flex;
   flex-direction: column;
 
+  &&& {
+    padding-inline: 0 !important;
+  }
+
```

Comment:
Seems to be connected to this:
![image](https://github.com/Cardinal-Cryptography/aleph-zero-signer/assets/35614528/14ffd198-1489-42fa-af8d-e71acaf079c9)

Reply:
Great catch. Yikes this is one of the components that has the scroll wrapper nested inside a container, breaking the layouts, but refactoring it is too dangerous atm 😬 I'm afraid I have to leave the padding reset, buy I'll add the white space back using margins.

Reply:
Btw I saw you found 2 cases like that so I went yet again trough all the scoll wrapper usages to check if there are other instances of that problem, but I think you found them all 🎖️